services/src/main/java/org/exoplatform/news/service/impl/NewsServiceImpl.java

```diff
@@ -200,6 +202,8 @@ public News getNewsById(String newsId, org.exoplatform.services.security.Identit
       news.setCanPublish(canPublishNews(currentIdentity));
       news.setCanArchive(canArchiveNews(currentIdentity, news.getAuthor()));
       return news;
+    } catch (IllegalAccessException e) {
+      throw new IllegalAccessException("User " + currentIdentity.getUserId()+ " attempt to access unauthorized news with id {}" + newsId);
     } catch (Exception e) {
```

Comment:
put this catch only on News news = getNewsById(newsId, editMode); with a log error